gen/host.go

```diff
@@ -303,23 +303,23 @@ func genPluginMethod(g *protogen.GeneratedFile, f *fileInfo, method *protogen.Me
 	g.P("data, err := request.MarshalVT()")
 	g.P(errorHandling)
 	g.P("dataSize := uint64(len(data))")
-	g.P(`if dataSize == 0 {
```

Comment:
before we return early without making requests to the plugin if dataSize==0... that's mistake